peerdroid/src/main/java/rdx/works/peerdroid/data/webrtc/wrappers/peerconnection/PeerConnectionSuspendingWrapper.kt

```diff
@@ -33,28 +32,42 @@ internal suspend fun PeerConnection.createSuspendingOffer(
 
             p0?.let {
                 continuation.resume(
-                    Result.Success(
+                    Result.success(
                         SessionDescriptionValue(p0.description)
                     )
                 )
             } ?: continuation.resume(
-                Result.Error("sessionDescription is null")
+                Result.failure(
+                    Throwable("sessionDescription is null")
```

Comment:
this might be topic for another conversation, but don't we need more meaningful exceptions instead of `Throwable`? cc @giannis-rdx

Reply:
well we can have more meaningful exceptions but we don't need to show them in the UI 🤷🏽‍♂️